pool/types.go

```diff
@@ -20,14 +20,20 @@
 
 package pool
 
-import "github.com/m3db/m3x/instrument"
+import (
+	"github.com/m3db/m3x/checked"
+	"github.com/m3db/m3x/instrument"
+)
 
-// Allocator allocates an object for a pool.
+// Allocator allocates an object for a pool
```

Comment:
Sure, will add back.  Unfortunately this is very inconsistently applied over all our projects.